data-processor/src/main/java/io/micronaut/data/processor/visitors/RepositoryTypeElementVisitor.java

```diff
@@ -449,6 +449,8 @@ private void processMethodInfo(MethodMatchContext methodMatchContext, MethodMatc
                 }
 
                 builder.member(AnnotationMetadata.VALUE_MEMBER, query);
```

Comment:
Need to populate `nativeQuery` annotation value later being checked in `DefaultStoredQuery`.